plugin/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ repositories {
 
 tasks.withType<KotlinCompile>() {
     kotlinOptions {
-        apiVersion = "1.3"
+        apiVersion = "1.4"
```

Comment:
matches apiVersion of ktlint.
Furthermore, we require at least gradle 6.8 that has a minimum requirement on kotlin 1.4.